nibabel/wrapstruct.py

```diff
@@ -536,8 +536,18 @@ def __str__(self):
 
         def _getter(obj, key):
             try:
-                return obj.get_value_label(key)
+                val = obj.get_value_label(key)
             except ValueError:
-                return obj[key]
+                val = obj[key]
+            # Convert a dtype('S1') numpy string to a Python str, if possible
+            if isinstance(val, np.ndarray) and val.dtype.type == np.bytes_:
+                try:
+                    tmp = val.tolist().decode()
+                    # Do not return unicode objects (Python 2)
+                    if isinstance(tmp, str):
```

Comment:
If you look at the doctest errors for nifti above, one of the problems is that text fields are printed as `b'foo'` when the tests are expecting `foo`. This resolves that issue and makes printing a little nicer in Python 3, without messing up Python 2.

The only difference is Python 2 will happily print non-unicode bytes, which Python 3 will still have to print as `b''` strings.